core/model/src/main/java/com/mifos/mobilewallet/model/domain/Account.kt

```diff
@@ -0,0 +1,18 @@
+package com.mifos.mobilewallet.model.domain
+
+import android.os.Parcel
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Account(
+    var image: String,
+    var name: String,
+    var number: String,
+    var balance: Double=0.0,
+    var id: Long=0L,
+    var productId: Long=0L,
+    var currency: Currency,
+) : Parcelable {
+    constructor() : this("", "", "", 0.0, 0L, 0L, Currency())
```

Comment:
We don't need constructor anymore, `Parcelize` annotation does everything for us.

Reply:
`Parcelize` does everything , but seems  do not provide us the empty constructor .
This is done to give the empty constructor .
Thanks .
So sticking with my pattern in all files .